test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpStatementTest.cs

```diff
@@ -148,6 +149,53 @@ public void FinallyClause()
                                ));
         }
 
+        public static TheoryData StaticUsingData
+        {
+            get
+            {
+                var factory = SpanFactory.CreateCsHtml();
+                Func<string, string, DirectiveBlock> createUsing = (code, import) =>
+                    new DirectiveBlock(
+                        factory.CodeTransition(),
+                        factory.Code(code)
+                            .AsNamespaceImport(import)
+                            .Accepts(AcceptedCharacters.AnyExceptNewline));
+
+                // document, expectedResult
+                return new TheoryData<string, DirectiveBlock>
+                {
+                    { "@using static", createUsing("using static", " static") },
+                    { "@using static    ", createUsing("using static    ", " static    ") },
+                    { "@using         static    ", createUsing("using         static    ", "         static    ") },
```

Comment:
Do the above 3 cases result in a Razor error? Or does it just cause a compilation error e.g. because the name of the static class (on the next line) is taken as HTML?